docs/develop/README.md

```diff
@@ -48,3 +48,10 @@ Depending on your use case, you may find these additional design patterns helpfu
 - [Currency Exchange](./recipes/currency-exchange.md)
 - [Multi-Debit, Multi-Credit Transfers](./recipes/multi-debit-credit-transfers.md)
 - [Closing Accounts](./recipes/close-account.md)
+
+## Get in Touch
```

Comment:
2 headings

## Ask the Community

Have questions about ...

## Dedicated Consultation

Would you like the TigerBeetle team to help you design your chart of accounts and leverage the power of TigerBeetle in your architecture?

Let us help you get it right. Contact our CEO, Joran Dirk Greef, at joran@tigerbeetle.com to set up a call.